website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,295 @@
+
+#### Manage Scope
+
+| Field      | Value                             |
+| ---------- | --------------------------------- |
+| Name       | Apple Business Manager ssf.manage |
+| Scope name | ssf.manage                        |
+| Expression | `return {}`                       |
+
+Finally, save the scope mapping and visually confirm the creation of all three scope mappings.
+
+### OAuth2/OpenID Provider
+
+From the authentik Admin interface, navigate to **Applications** -> **Providers** and press **Create**.
+
+:::tip{title="Keep your text editor ready"}
+
+authentik will automatically generate the _Client ID_ and _Client Secret_ values for the new provider.
+
+You'll need these values when configuring Apple Business Manager.
+
+:::
+
+From the creation wizard, select **OAuth2/OpenID Provider** and continue with the following form values:
+
+| Field                 | Value                                               |
+| --------------------- | --------------------------------------------------- |
+| Name                  | Apple Business Manager                              |
+| Authorization Flow    | `default-provider-authorization-implicit-consent`   |
+| Redirect URIs/Origins | `https://gsa-ws.apple.com/grandslam/GsService2/acs` |
+
+Under **Advanced protocol settings**, make sure to add the newly created scope mappings to the provider, as well as the following additional scopes:
+
+1. `ssf.manage`
+2. `ssf.read`
+3. `profile`
+4. `authentik default OAuth Mapping: OpenID 'email'`
+5. `authentik default OAuth Mapping: OpenID 'openid'`
+6. `authentik default OAuth Mapping: OpenID 'offline_access'`
+
+Visually confirm that you've copied the **Client ID** and **Client Secret** before pressing **Finish**. You'll need these values when configuring Apple Business Manager. Save the provider and confirm that the new provider is listed in the provider overview.
+
+:::tip{title="OpenID Configuration URL"}
+
+Before leaving the provider overview page, make sure to copy the **OpenID Configuration URL** value to your text editor. You'll need this when configuring Apple Business Manager.
+
+You can always find your provider's generated URLs by navigating to **Providers**, then selecting the provider by name. The URLs are listed under the **Overview** tab.
+
+:::
+
+### Shared Signals Framework Provider
+
+While the OAuth2/OpenID Provider handles the authentication flow, you'll need to create a Shared Signals Framework Provider to handle the backchannel communication between authentik and Apple Business Manager.
+
+Once more from the authentik Admin interface, navigate to **Applications** -> **Providers** and press **Create**. This time, select **Shared Signals Framework Provider** and continue with the following form values:
+
+| Field | Value                      |
+| ----- | -------------------------- |
+| Name  | apple-business-manager-ssf |
+
+Press **Finish** to create the Shared Signals Framework Provider.
```

Comment:
```suggestion
Click **Finish** to create the Shared Signals Framework Provider.
```

Reply:
There are more values than just the **Name** to define, when creating an SSF provider, right? I think it would be best to simply link to the docs on how to create an SSF provider, instead of duplicating those steps here. Thoughts?